libyul/backends/evm/EVMMetrics.cpp

```diff
@@ -49,7 +49,7 @@ size_t GasMeter::instructionCosts(evmasm::Instruction _instruction) const
 
 size_t GasMeter::combineCosts(std::pair<size_t, size_t> _costs) const
 {
-	return _costs.first * m_runs + _costs.second;
+	return m_isCreation ? (_costs.first + _costs.second) : (_costs.first * m_runs + _costs.second);
```

Comment:
The same can be achieved by forcing `m_runs` to 1 in the constructor. Do you think that would be a better solution?

Reply:
I forced `m_runs` to `1` for creation context.